push.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+BRANCH=master
+REMOTE=origin
```

Comment:
This should also be:
```diff
+FOLDER=$1
+EXPORT_URL='https://www.google.com/maps/d/u/0/kml?mid=180u1IkUjtjpdJWnIC0AxTKSiqK4G6Pez'
+SAVE_TO=${FOLDER:-.}/$(date '+%y%m%d_%H%M').kmz
```

Reply:
Thanks. Should be updated now.
My fat-finger conflict resolving broke this.